test/TestSuppressions.cs

```diff
@@ -0,0 +1,5 @@
+﻿// Suppressions file for test projects.
```

Comment:
Is this better than `NoWarn` on SA1600? (I don't have a real preference, just curious if you find one way to be better than the other.)

In other projects I've done, I actually have two different rulesets - one for production code, one for tests. The test one is less stringent. Maybe something to consider in the future.

Reply:
I just did it like this so I could share it, but perhaps it's better to put the NoWarn in a Directory.props file in the tests folder?

Reply:
Meh. It's fine for now. Some later point I might put two different ruleset files in there for better control, but I'm not going to be a stickler over it.

Reply:
Whoops, too late!